odd_collector/adapters/mongodb/adapter.py

```diff
@@ -30,103 +26,17 @@ def get_data_entities(self) -> List[DataEntity]:
         an odd list of data entities
         """
         try:
-            self.connect()
-            schemas = self.retrive_schemas()
-
+            schemas = self.__mongo_repository.retrieve_schemas()
             return map_collection(self.__oddrn_generator, schemas, self.__database)
+        except DBException as db_e:
+            logging.error("Failed retrieve data from Database")
         except Exception as e:
             logging.error("Failed to load metadata for tables")
```

Comment:
You can write it as `logging.error(..., exc_info=True)` and  delete `logging.exception` below

Reply:
replaced logging.exception(e) with exc_info=True in the commit [a6be551](https://github.com/opendatadiscovery/odd-collector/pull/70/commits/a6be551fab87fd7ebcd63301a9c352f4068f7dcc)